nodejs/__test__/table.test.ts

```diff
@@ -448,6 +455,71 @@ describe("When creating an index", () => {
   });
   afterEach(() => tmpDir.removeCallback());
 
+  it('tests "offset" for "vectorSearch"', async () => {
+    const vectorSearch = (offset: number, limit = 1) =>
+      tbl
+        .vectorSearch(queryVec)
+        .limit(limit)
+        .offset(offset)
+        .select("id")
+        .toArray();
+    const rowsFirstTwo = await vectorSearch(0, 2);
+    expect(rowsFirstTwo.length).toBe(2);
+    const rowsFirst = await vectorSearch(0);
+    expect(rowsFirst.length).toBe(1);
+    const rowsSecond = await vectorSearch(1);
+    expect(rowsSecond.length).toBe(1);
+    expect([rowsFirst[0].id, rowsSecond[0]?.id]).toEqual(
+      rowsFirstTwo.map((v) => v.id),
+    );
+  });
+
+  it("runs full text search w/ initially empty table", async () => {
+    const db = await connect(tmpDir.name);
+    const schema = new arrow.Schema([
+      new arrow.Field("text", new arrow.Utf8()),
+    ]);
+    const table = await db.createEmptyTable(
+      "full-text-search-with-initially-empty-table",
+      schema,
+      {
+        mode: "overwrite",
+        existOk: true,
+      },
+    );
+    const data: { text: string }[] = [
+      { text: "hello world" },
+      { text: "goodbye world" },
+    ];
+    await table.add([data[0]]); // add multiple rows (i.e. do a batch) in real life; adding rows one by one is just for testing here
+    await table.createIndex("text", { config: Index.fts() }); // index must be created after there's at least one row; otherwise, an index is automatically removed when the first row is added
+    await table.add([data[1]]); // add multiple rows (i.e. do a batch) in real life; adding rows one by one is just for testing here
+
+    const runSearch = () =>
+      table.search("world", "fts", "text").fastSearch().toArray();
+
+    const results1 = await runSearch();
+    /** @todo the commented out test must pass */
+    // expect(results1.length).toBe(1); // since we use ".fastSearch", the "data[1]" row doesn't exist
+    /** @todo this must fail */ expect(
```

Comment:
@wjones127 `.fastSearch()` doesn't work. It doesn't search through indexed data only.